### DISCUSSION TITLE
你好，请问你能给展开的姿势部件列表专门做一个滚动条吗？

### DISCUSSION THREAD

**Discussion by aomaoyyw** [huchenlei/sd-webui-openpose-editor#25] [Ideas]
你好，请问你能给展开的姿势部件列表专门做一个滚动条吗？（或者索性给所有左边的控件做一个滚动条？）
我发现当这个列表展开时会变得非常非常的长，以至于当我去翻找相应部件时，
右边openpose的图已经被滚动上去看不到了，我不得不在openpose的图和姿势部件列表之间来回滚动页面。
比如我需要确定没显示的手指是哪个finger时，我就需要来回翻找，这非常的不方便。
再次感谢你的插件。
Hi, can you make a scroll bar specifically for the expanded list of pose parts? (or simply make a scroll bar for all left controls?)
I found that when the list was expanded, it became very, very long, so that when I went to find the corresponding parts, the openpose figure was no longer visible, and I had to scroll back and forth between the openpose figure and the pose part list page. 
For example, I can't be sure which finger of which hand in the picture the finger2 that is not displayed corresponds to. I need to search back and forth, which is very inconvenient. 
Thanks again for the plugin.
<img width="910" alt="屏幕截图 2023-06-16 020956" src="https://github.com/huchenlei/sd-webui-openpose-editor/assets/33321738/ff522184-9108-4086-a9b0-119522f68ca9">

**Comment by huchenlei** [comment]
Sure. That seems a good UI improvement. Going to address it in the next release.

**Comment by huchenlei** [comment]
Fixed in v0.1.7
![Screenshot (136)](https://github.com/huchenlei/sd-webui-openpose-editor/assets/20929282/c41ca4f2-9215-43d6-b30c-954ee431a069)

**Comment by aomaoyyw** [comment]
非常好用，感谢
